concordium-consensus/src/Concordium/GlobalState/BlockState.hs

```diff
@@ -857,19 +864,19 @@ class (BlockStateQuery m) => BlockStateOperations m where
     -- set of passive delegators. In each case, the lists are ordered in ascending Id order,
     -- with no duplicates.
     bsoGetActiveBakersAndDelegators ::
-        (SupportsDelegation (MPV m)) =>
+        (AVSupportsDelegation (AccountVersionFor (MPV m))) =>
         UpdatableBlockState m ->
         m ([ActiveBakerInfo m], [ActiveDelegatorInfo])
 
     -- |Get the bakers for the epoch in which the block was baked.
     bsoGetCurrentEpochBakers :: UpdatableBlockState m -> m FullBakers
 
     -- |Get the bakers for the epoch in which the block was baked, together with their commission rates.
-    bsoGetCurrentEpochFullBakersEx :: (SupportsDelegation (MPV m)) => UpdatableBlockState m -> m FullBakersEx
+    bsoGetCurrentEpochFullBakersEx :: (AVSupportsDelegation (AccountVersionFor (MPV m))) => UpdatableBlockState m -> m FullBakersEx
```

Comment:
Maybe we should introduce a type `SupportsDelegationPV = AVSupportsDelegation (AccountVersionFor pv)` in `base`